docs/architecture/grpc-for-wcf-developers/why-grpc.md

```diff
@@ -1,11 +1,41 @@
 ---
 title: Why gRPC is recommended for WCF developers - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: A discussion of why gRPC is a good fit for WCF developers looking to migrate to modern architectures and platforms.
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # Why gRPC is recommended for WCF developers
 
+Before diving deeply into the language and techniques of gRPC, it's worth pulling together some ideas about why gRPC is the right solution for WCF developers who want to migrate to .NET Core, given there are alternatives available.
+
+## Similarity to WCF
+
+Although its implementation and approach is different, the actual experience of developing and consuming services with gRPC should be very intuitive for WCF developers. The underlying goal of making it possible to code as though the client and server were on the same platform, without needing to worry about networking, is the same. Both platforms share the principle of declaring and then implementing an interface, even though the process for declaring that interface is different. And as you'll see in chapter 5, the different types of RPC calls supported by gRPC map very well to the different bindings available to WCF services.
+
+## Benefits of gRPC
+
+Additional reasons why gRPC stands above other solutions are:
+
+### Performance
+
+As already discussed, using HTTP/2 rather than HTTP/1.1 removes the requirement for human-readability code and instead use the smaller faster binary protocol. This is more efficient for computers to parse. HTTP/2 also supports multiplexing requests over a single connection enabling responses to be sent as soon as they're ready without the need to wait in a queue. Fewer resources are needed when using gRPC, which makes it a good solution to use for mobile devices and over slower networks. [Mark to check]
```

Comment:
```suggestion
As already discussed, using HTTP/2 rather than HTTP/1.1 removes the requirement for human-readable code and instead uses the smaller faster binary protocol. This is more efficient for computers to parse. HTTP/2 also supports multiplexing requests over a single connection enabling responses to be sent as soon as they're ready without the need to wait in a queue. Fewer resources are needed when using gRPC, which makes it a good solution to use for mobile devices and over slower networks. [Mark to check]
```

Reply:
Don't forget to remove [Mark to check]